src/client/players.py

```diff
@@ -45,19 +45,25 @@ def isFriend(self, id):
         '''
         Convenience function for other modules to inquire about a user's friendliness.
         '''
+        if id in self._logins:
+            id = self._logins[id].id
```

Comment:
Wait, what? If logins is indexed by user id, what's the point of using it to look up user id?
Are there two different coexisting ID schemes or something (if so, why?)